### PR TITLE
Ensure testgrid config contains parseable emails

### PR DESCRIPTION
**Pull request by spiffxp**
Try to avoid another https://github.com/kubernetes/test-infra/pull/12526

### CODE REVIEW

**Comment by spiffxp** [comment]
Based on local testing this will fail if given:
- `alert_mail_to_addresses: `
- `alert_mail_to_addresses: foo`
- `alert_mail_to_addresses: foo@bar.com,`

File: testgrid/cmd/configurator/config_test.go [hunk 1]
```diff
@@ -479,3 +481,20 @@ func TestReleaseBlockingJobsMustHaveTestgridDescriptions(t *testing.T) {
 		}
 	}
 }
+
+func TestNoEmpyMailToAddresses(t *testing.T) {
+	for _, dashboard := range cfg.Dashboards {
+		for _, dashboardtab := range dashboard.DashboardTab {
+			intro := fmt.Sprintf("dashboard_tab %v/%v", dashboard.Name, dashboardtab.Name)
+			if dashboardtab.AlertOptions != nil {
+				mails := strings.Split(dashboardtab.AlertOptions.AlertMailToAddresses, ",")
```
Comment (michelle192837) [inline testgrid/cmd/configurator/config_test.go]: Consider trimming leading/trailing whitespace as well.

**Comment by michelle192837** [comment]
Just to note, I have a change to ignore trailing commas/empty addresses, but this is good practice anyways and redundancy can't hurt!

File: testgrid/cmd/configurator/config_test.go (hunk 1 above)
Comment (spiffxp) [reply testgrid/cmd/configurator/config_test.go]: `mail` is happy with leading/trailing whitespace, do you want this test to fail if there is leading/trailing whitespace?

Comment (michelle192837) [reply testgrid/cmd/configurator/config_test.go]: Hmm. Welp this is handled already by TestGrid code when the addresses are used, so either way.

**Comment by michelle192837** [comment]
/lgtm
/hold

Feel free to unhold at will!

**Pull request merged**